src/main/proto/cosmas.proto

```diff
@@ -37,9 +37,34 @@ message CreateVersionRequest {
 message CreateVersionResponse {
 }
 
+message CreateHistoryOfVersionRequest {
+    string user = 1;
+    string fileId = 2;
+    string patch = 3;
+    int64 timeStamp = 4;
+}
+
+message CreateHistoryOfVersionResponse {
+}
+
+message GetHistoryOfVersionRequest {

```

Comment:
There were no requirement to add get request in the issue (it talks about sending patches only), but since you added it, what in your opinion is the scenario of using the patch history? How will it look like from user perspective and how will it be translated to RPC requests? Do you probably need to change the request and response to fit the access pattern better?

Reply:
But how I can test add request if I haven't any get requests?

Reply:
You don't have to use public grpc interface in the test. If you test the implementation, you can add methods which are not exposed in any interface but which can help you with accessing object state.